mcs/build/tests.make

```diff
@@ -270,6 +270,8 @@ XTEST_HARNESS_FLAGS := -noappdomain -noshadow -parallel none -nunit TestResult-$
 XTEST_TRAIT := -notrait category=failing -notrait category=nonmonotests -notrait Benchmark=true -notrait category=outerloop
 TEST_MONO_PATH := $(TEST_MONO_PATH)$(PLATFORM_PATH_SEPARATOR)$(XTEST_HARNESS_PATH)
 
+TEST_MONO_PATH := $(XTEST_HARNESS_PATH):$(TEST_MONO_PATH)
```

Comment:
`:` doesn't work on Windows as a PATH separator, you can use `$(PLATFORM_PATH_SEPARATOR)` instead. Not sure why you need this though given the line above? merge conflict?

Reply:
Yeah this change should not be needed if you rebased on top of master

Reply:
Fixed.